teamA/crud-express/heeyeon/app.js

```diff
@@ -0,0 +1,178 @@
+require('dotenv').config();
+
+const http = require('http');
+
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+const { DataSource } = require('typeorm');
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPOPRM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+})
+
+myDataSource.initialize()
+    .then(() => {
+        console.log('Data Source has been initialized!')
+    });
+
+const app = express()
+
+app.use(express.json());
+app.use(morgan('tiny'));
+
+// health check
+app.get('/ping', cors(), function (req, res, next) {

```

Comment:
cors 미들웨어를 하나의 http 요청 엔드포인트에만 작성한다면, 해당 요청에 대해서만 cors 정책이 적용되어 다른 http 통신 간 cors error를 발생시킬 수 있습니다!

app 객체에 cors 미들웨어를 적용하여 전체 엔드포인트 요청에 대해 cors 정책을 허용하도록 수정해주세요!

```suggestion
app.use(cors());

app.get('/ping', function (req, res) {

```